### ISSUE TITLE
String cannot represent a non string value

### ISSUE THREAD

**Issue by sebasptsch** [OpenSaasAU/keystone-nextjs-auth#43]
Hi, I've just begun to use your package and added a github authentication provider but, whenever I complete the authentication flow I get this error. 
```
http://localhost:3000/admin/api/auth/error?error=Variable%20%22%24where%22%20got%20invalid%20value%2032213671%20at%20%22where.subjectId.equals%22%3B%20String%20cannot%20represent%20a%20non%20string%20value%3A%2032213671
```
I'm running the latest version of this package (14.1.1) and version 26.1.0 of `@keystone-next/keystone`

Link to Github user API reference:
https://docs.github.com/en/rest/reference/users

**Comment by sebasptsch** [comment]
Fixed by changing the subjectId field to integer instead of text.

**Comment by sebasptsch** [comment]
Sorry, still having this issue but in a different part of the process.
```
Create User
[next-auth][error][oauth_callback_handler_error] 
https://next-auth.js.org/errors#oauth_callback_handler_error GraphQLError [Object]: Variable "$where" got invalid value "32213671" at "where.subjectId.equals"; Int cannot represent non-integer value: "32213671"
    at C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\execution\values.js:116:15
    at coerceInputValueImpl (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\utilities\coerceInputValue.js:131:9)
    at coerceInputValueImpl (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\utilities\coerceInputValue.js:105:34)
    at coerceInputValueImpl (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\utilities\coerceInputValue.js:105:34)
    at coerceInputValueImpl (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\utilities\coerceInputValue.js:54:14)
    at coerceInputValue (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\utilities\coerceInputValue.js:37:10)
    at _loop (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\execution\values.js:109:69)
    at coerceVariableValues (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\execution\values.js:121:16)
    at getVariableValues (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\execution\values.js:50:19)
    at buildExecutionContext (C:\Users\Sebas\Gits\NextCMS\node_modules\graphql\execution\execute.js:203:61)
```

**Comment by borisno2** [comment]
Hi @sebasptsch
Thanks for letting me know about this issue with GitHub provider. I can see where the issue is, I'll have a look when I am back at my desk and see what I can do...
Thanks,
Josh

**Comment by borisno2** [comment]
Hi @sebasptsch 
I just released version `15.0.0` which should allow for providers like GitHub that use numbers for the subjectId. Let me know if you run into any other issues with GitHub.
Thanks,
Josh
